backend/qemu.pm

```diff
@@ -264,6 +279,20 @@ sub start_qemu() {
         symlink($i, "$basedir/l$i") or die "$!\n";
     }
 
+    if ($vars->{NICTYPE} eq "vde") {
+        if (system('vde_switch', '-d', '-s', "/tmp/openqa_vde$vars->{NIC_VLAN}.ctl") == 0) {
```

Comment:
Scheduler should make sure that it is not undef. If it is undef anyway, then connecting to unnumbered vde swith is IMHO a reasonable fallback. OK, I could fix the warning.

"/tmp/openqa_vde$vars->{NIC_VLAN}.ctl" is a directory. AFAIK creating directories under /tmp is safe.